### DISCUSSION TITLE
Grouping in the taskbar

### DISCUSSION THREAD

**Discussion by Hottihotte** [valinet/ExplorerPatcher#3875] [General]
How can you move different files of the same "type" (for example 2 different text files (Natepad) separately in the taskbar?
I know that 7+Taskbartweaker can do it and also a suitable Windhawk mod, but does it also work with ExplorerPatcher?

**Comment by pyrates999** [comment]
when you have icons grouped together?

**Comment by Amrsatrio** [comment]
What's the "suitable Windhawk mod"? Some Windhawk mods support EP's taskbar.

**Reply by Hottihotte** [reply]
> What's the "suitable Windhawk mod"? Some Windhawk mods support EP's taskbar.

[https://windhawk.net/mods/taskbar-grouping](url)
![uLITliK](https://github.com/user-attachments/assets/40c7632d-6f95-444a-82f8-8b14aef9d0e7)
Causes a separate button to be created on the taskbar for each new window. For example, notice the two separate buttons for Notepad on the screenshot
But I would be reluctant to run two tools.

Small addendum
The mod only works if you set the taskbar style in ExplorerPatcher to Windows 11 (standard) or Windows 10 (which has no effect).
The mod does not work with Windows 10 (Explorerpatcher).

**Reply by pyrates999** [reply]
have you installed the latest release of EP? https://github.com/valinet/ExplorerPatcher/releases

**Reply by Hottihotte** [reply]
> have you installed the latest release of EP? https://github.com/valinet/ExplorerPatcher/releases

Yes

**Reply by pyrates999** [reply]
Please post the version of EP you have installed.

**Reply by Hottihotte** [reply]
Version
![Screenshot 2024-11-05 013958](https://github.com/user-attachments/assets/f8697275-3177-4709-ab4d-d2e95013a67d)

With taskbar style "Windows 10 (ExplorerPatcher)"

https://github.com/user-attachments/assets/68446af4-d4d3-4fe4-a1ca-be3558fa7177

With taskbar style "Windows 10".but this has no effect on the taskbar of Windows 11 24h2.

https://github.com/user-attachments/assets/988d5673-3a03-45e0-adfa-71adce379208

**Reply by pyrates999** [reply]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

**Reply by Hottihotte** [reply]
> Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**. We no longer offer more support on the stock Windows 10 taskbar.

Unfortunately it doesn't work or I'm doing something wrong

https://github.com/user-attachments/assets/92bc78f3-786a-4bac-9b19-0ee26202b6e9


https://github.com/user-attachments/assets/516075b5-8352-4d85-8d1f-156bda0449da

The two settings work, but the taskbar style "Windows 11 Standard" and "Windows 10" is no longer supported by EP and with the taskbar style "Windows 10 ExplorerPatcher" the whole thing looks like this, please pay attention to the folders in the taskbar.


https://github.com/user-attachments/assets/f77c0195-6199-4306-90bf-4801ac2921e1

It is no longer possible to move different files of the same application separately.

**Reply by pyrates999** [reply]
I understand what you want now.  Please wait for a developer to respond.

**Reply by Amrsatrio** [reply]
> Note: To customize the old taskbar on Windows 11 (if using ExplorerPatcher or a similar tool), enable the relevant option in the mod's settings.

Have you?

**Reply by Hottihotte** [reply]
> > Note: To customize the old taskbar on Windows 11 (if using ExplorerPatcher or a similar tool), enable the relevant option in the mod's settings.
> 
> Have you?

I have to apologize, I missed that.
Now it works perfectly
